src/routes/likernft/book/store.ts

```diff
@@ -361,7 +370,10 @@ router.put(['/:classId/price/:priceIndex', '/class/:classId/price/:priceIndex'],
     }
 
     prices[priceIndex] = newPriceInfo;
-    const enableCustomMessagePage = prices.some((p) => !p.isAutoDeliver);
+    const enableCustomMessagePage = prices.some((p) => !p.isAutoDeliver)
+      || docEnableCustomMessagePage
+      || enableSignatureImage
+      || !!signedMessageText;
```

Comment:
align with the code above
```suggestion
    const enableCustomMessagePage = docEnableCustomMessagePage
      || enableSignatureImage
      || !!signedMessageText
      || prices.some((p) => !p.isAutoDeliver);
```